duradmin/src/main/webapp/WEB-INF/jsp/spaces/spaces-manager.jsp

```diff
@@ -120,7 +120,7 @@
         padding: 2px;
         color: #fff;
       }
-      
+
```

Comment:
This formatting change probably should be reverted or at least added as a separate formatting themed commit since it is not related to duracloud-1214 (as far as I can tell).